app/controllers/users_controller.rb

```diff
@@ -1,7 +1,9 @@
 class UsersController < ApplicationController
 
-  def list
-    @users = User.all
+  def create
+    @user  = User.create_from_params(params[:user])
+    @beard = Beard.create(:user_id => @user.id,
```

Comment:
Normally if you wanna be strict with object orientation you wouldn't assign by `#id`. That will become `user: @user` assignment by `#id` is the concern of the persistence layer and not you application.